bot/cogs/snakes.py

```diff
@@ -0,0 +1,1208 @@
+                    messages.append(message.content)
+
+                message = get_random_long_message(messages)
+
+            # Set the avatar
+            if user.avatar is not None:
+                avatar = f"https://cdn.discordapp.com/avatars/{user.id}/{user.avatar}"
+            else:
+                avatar = (
+                    "https://img00.deviantart.net/eee3/i/2017/168/3/4/"
+                    "discord__app__avatar_rev1_by_nodeviantarthere-dbd2tp9.png"
+                )
+
+            # Build and send the embed
+            embed.set_author(
+                name=f"{user.name}#{user.discriminator}",
+                icon_url=avatar,
+            )
+            embed.description = f"*{self._snakify(message)}*"
+
+            await ctx.channel.send(embed=embed)
+
+    @command(name="snakes.video()", aliases=["snakes.video", "snakes.get_video()", "snakes.get_video"])
+    async def video(self, ctx: Context, search: str = None):
+        """
+        Gets a YouTube video about snakes
+        :param name: Optional, a name of a snake. Used to search for videos with that name
+        :param ctx: Context object passed from discord.py
+
+        Written by Andrew and Prithaj.
+        """
+
+        # Are we searching for anything specific?
+        if search:
+            query = search + ' snake'
+        else:
+            snake = await self._get_snake_name()
+            query = snake['name']
+
+        # Build the URL and make the request
+        url = f'https://www.googleapis.com/youtube/v3/search'
+        response = await self.bot.http_session.get(
+            url,
+            params={
+                "part": "snippet",
+                "q": urllib.parse.quote(query),
+                "type": "video",
+                "key": YOUTUBE_API_KEY
+            }
+        )
+        response = await response.json()
+        data = response['items']
+
+        # Send the user a video
+        if len(data) > 0:
+            num = random.randint(0, len(data) - 1)
+            youtube_base_url = 'https://www.youtube.com/watch?v='
+            await ctx.channel.send(
+                content=f"{youtube_base_url}{data[num]['id']['videoId']}"
+            )
+        else:
+            log.warning(f"CRITICAL ERROR: YouTube API returned {response}")
+
+    @command(name="snakes.zen()", aliases=["zen"])
+    async def zen(self, ctx: Context):
+        """
+        Gets a random quote from the Zen of Python.
+
+        Written by Prithaj and Andrew.
+        Modified by lemon.
+        """
+
+        embed = Embed(
+            title="Zzzen of Pythhon",
+            color=SNAKE_COLOR
+        )
+
+        # Get the zen quote and snakify it
+        zen_quote = random.choice(ZEN.splitlines())
+        zen_quote = self._snakify(zen_quote)
+
+        # Embed and send
+        embed.description = zen_quote
+        await ctx.channel.send(
+            embed=embed
+        )
+    # endregion
+
+    @snake_card.error
```

Comment:
This doesn't have a region, less of a requirement but could it go in a region such as "Error Handlers" or something similar.

Reply:
a bit weird to have a region for a single method, but I'll do it anyway.